cst438_gradebook/src/main/java/com/cst438/controllers/AssignmentController.java

```diff
@@ -0,0 +1,63 @@
+package com.cst438.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import com.cst438.domain.Assignment;
+import com.cst438.domain.AssignmentListDTO;
+import com.cst438.domain.AssignmentRepository;
+import com.cst438.domain.Course;
+import com.cst438.domain.CourseRepository;
+
+//import java.util.Arraylist;
+
+@RestController 
+public class AssignmentController 
+{
```

Comment:
assignment controller looks good on indentation and commented well.

Reply:
only suggestion I can think of is maybe look into java Optional objects to handle null pointer exceptions a little better, overall looks great!